content/en/external-automation/webhook-approval.md

```diff
@@ -10,7 +10,46 @@ tags: ["Webhooks", "GitHub", "Automation"]
 
 ## {{% heading "prereq" %}}
 
-You are familiar with [using external automation]({{< ref "webhooks/_index.md" >}}) with CD-as-a-Service 
+You have read {{< linkWithTitle "external-automation/overview.md" >}}.
+
+## Requirements for your webhook and callback
```

Comment:
I believe How to trigger Call back should be after how to trigger webhook in deployment file. 
1. It makes sense in the flow - Configure Webhook, Call your webhook and then respond with a callback
2. Since callbackUri is passed from the deployment file, it is important that we showcase that before introducing it magically.